addons/stock/views/stock_picking_views.xml

```diff
@@ -272,7 +273,7 @@
                                     <field name="lot_id" groups="stock.group_production_lot" attrs="{'column_invisible': [('parent.show_lots_text', '=', True)], 'invisible': [('lots_visible', '=', False)], 'readonly': [('in_entire_package', '=', True)]}" domain="[('product_id', '=', product_id)]" context="{'default_product_id': product_id}"/>
                                     <field name="lot_name" groups="stock.group_production_lot" attrs="{'column_invisible': [('parent.show_lots_text', '=', False)], 'invisible': [('lots_visible', '=', False)], 'readonly': [('in_entire_package', '=', True)]}" context="{'default_product_id': product_id}"/>
                                     <field name="is_initial_demand_editable" invisible="1"/>
-                                    <field name="product_uom_qty" attrs="{'readonly': ['|', ('is_initial_demand_editable', '=', False), ('in_entire_package', '=', True)], 'required': [('product_id', '!=', False)]}"/>
+                                    <field name="product_uom_qty" attrs="{'required': [('product_id', '!=', False)]}" readonly="1"/>
```

Comment:
Not tested, but that seems odd. The field is required, but always read-only. Couldn't this lead to a deadlock?

Reply:
It represent the reserved quantity and should never be set by the user. The default value set in the view is 0 and is the expected one. I've just retested it and it doesn't seems to cause any trouble.

Reply:
OK, so is there any reason to keep it required? `product_id` is required anyway. By the way, since the reserved quantity should never be set by the user, maybe the `readonly` attribute should be directly on the field.